docs/application/tizen-studio/common-tools/device-manager.md

```diff
@@ -193,15 +201,15 @@ You can set the following settings for the Device Manager:
 
     This is the maximum wait time for push and pull operations in the **File Explorer** view. To set the timeout value, select the **Device** tab, change the value, and click **OK**.
 
-    > **Note**  
-    > Only values between 30 000 and 600 000 (milliseconds) are allowed.
+    > [!NOTE]   
+    > Only values between 30,000 and 600,000 (milliseconds) are allowed.
 
 - Log buffer size
 
     This is the maximum amount of log entries to store in the log table. To set the buffer size, select the **Log** tab, change the value, and click **OK**.
 
-    > **Note**  
-    > Only values between 5 000 and 100 000 (lines) are allowed.
+    > [!NOTE]   
+    > Only values between 5000 and 100,000 (lines) are allowed.
```

Comment:
Question. In case of "5000", don't we need to use form of "5,000"?

Reply:
Thank you, I will change it to 5,000.

Reply:
Done